tests/test_pandas_utils.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd
+import numpy as np
+import logging
+
+from sklego.pandas_utils import log_step
+
+logging.basicConfig(level=logging.INFO)
+
+
+def test_logging(caplog):
```

Comment:
maybe add a fixture for a more general dataframe 

- maybe add a test that also demonstrates that the number of rows actually decrease. 
- i might also enjoy a test that shows the `**kwargs` appear